lib/cadet/assessments/assessments.ex

```diff
@@ -481,6 +499,33 @@ defmodule Cadet.Assessments do
     Question.changeset(%Question{}, params_with_assessment_id)
   end
 
+  def insert_voting(voting_params, assessment_id) do
+    question =
+      Question
+      |> where(assessment_id: ^assessment_id)
+      # voting assessment only has one question
+      |> first()
+      |> Repo.one()
+
+    changesets =
+      Enum.map(voting_params, fn voting_entry ->
+        build_voting_submission_changeset_for_assessment_id(voting_entry.voting, question.id)
+      end)
+
+    changesets
+    |> Enum.with_index()
+    |> Enum.reduce(Multi.new(), fn {changeset, index}, multi ->
+      Multi.insert(multi, Integer.to_string(index), changeset)
+    end)
+    |> Repo.transaction()
+  end
+
+  defp build_voting_submission_changeset_for_assessment_id(params, question_id)
```

Comment:
The function name doesn't match the parameters, is this supposed to be `build_voting_submission_changeset_for_question_id`?